README.md

```diff
@@ -3,22 +3,42 @@
 [![Board Status](https://dev.azure.com/tc-tibo/4dd797bf-468b-4734-809d-7a5d7e801e2d/98f281cb-b20c-4b34-9384-beeaf67bef78/_apis/work/boardbadge/04d8c3b1-ac45-422d-9c05-c39db02d017d?columnOptions=2&columns=To%20Do,Doing,Done)](https://dev.azure.com/tc-tibo/4dd797bf-468b-4734-809d-7a5d7e801e2d/_boards/board/t/98f281cb-b20c-4b34-9384-beeaf67bef78/Microsoft.RequirementCategory/)
 
 # Introduction 
-TODO: Give a short introduction of your project. Let this section explain the objectives or the motivation behind this project. 
+
+This is the prototype for TC's Seafarer Credentials Project. It is written in Blazor and is strictly for development purposes. Actual development will be hosted in TC's official Github home.
 
 # Getting Started
-TODO: Guide users through getting your code up and running on their own system. In this section you can talk about:
+
 1.	Installation process
+
+To install this project, clone locally, open https://github.com/tc-tibo/SeaCertsBlazor/blob/master/CDTS%20Blazor.sln file in Visual Studio, and then publish to your desired location.
```

Comment:
Could probably remove this line. I don't think it's important for the installation instructions.

Reply:
You don't need to open sln file? I know ZERO about Blazor

Reply:
sln files are solution generated files for Visual Studio has nothing to do with blazor. I created a review to reword the sentence.

Reply:
```suggestion
To run this project locally from VisualStudio 2019, clone repository, open .sln file in Visual Studio, and then choose Debug -> Start Debugging, or F5 on the keyboard.
```